Automattic/vip-cli
# Fetch image tags from Docker Hub in dev environment

## Description

This PR adds the ability for the `dev-environment create` command to fetch the latest available version of the [WP VIP WordPress image available in Docker Hub](https://hub.docker.com/r/wpvipdev/wordpress).

We are updating the command by doing a `GET` request to Docker Hub each time the user creates a new environment to fetch the latest available tag. 

## Steps to Test

1. Check out PR.
2. Run `npm run build`
3. Run `./dist/bin/vip-dev-environment-create --slug test`
4. You should see a `.lando.yml` file created with the latest WP version (at the moment of this PR, `5.7.1`.
